raku/t/01-load.rakutest

```diff
@@ -14,3 +16,5 @@ foo:: 1..10
 
 my YAMLScript $ys.=new;
 is-deeply %wanted, $ys.load($program);
+
+ok try { $ys.DESTROY; True } // False;
```

Comment:
How does this work?

If the call to `graal_create_isolate` in `DESTROY` can die, then when are we going to get to the `warn` in [line 29](https://github.com/yaml/yamlscript/pull/26/files?diff=split&w=0#diff-a29edd0d71c3aeacbd4f8959c88a1cf20f9cb37c5f2a98aad0e06227dd6dc46dR29)?

If it won't die, and we _will_ hit the `warn` (if the return value is false), then we don't need the `try`, and this test could be `ok $ys.DESTROY` (assuming we are happy to rely on `warn` returning a falsy value).

Or am I reading things incorrectly?

Reply:
I guess my question is what is dying inside your `DESTROY` that makes the safeguard meaningful. Like, when it dies, what makes it die? Because from reading the code, it doesn't look to me like it should be dying (even if it is called twice)